### PR TITLE
Storing UUID locally at Gateway to be used by other agents

### PR DESCRIPTION
**Pull request by KohliDev**
Storing UUID locally at the gateway in uuid.ini file (location /etc/liota/conf) to be used by other agents.

### CODE REVIEW

**Comment by GregBollella** [comment]
Ship it!

File: liota/dcc/vrops.py [hunk 1]
```diff
@@ -39,11 +39,12 @@
 import sys
 import threading
 from time import timezone
+import ConfigParser
 
 from liota.dcc.dcc_base import DataCenterComponent
 from helix_protocol import HelixProtocol
 from liota.core.metric_handler import Metric
-from liota.utilities.utility import getUTCmillis
+from liota.utilities.utility import getUTCmillis, findLiotaConfigFullPath
```
Comment (pmasrani) [inline liota/dcc/vrops.py]: It is not apparent that findLiotaConfigFullPath is a class. Generally, class names should be UpperCaseCamelCase. Also, class names are generally nouns and method names are actions like 'find'. Can you please change that?

File: liota/dcc/vrops.py [hunk 2]
```diff
@@ -117,6 +118,28 @@ def on_receive_safe(msg):
...
+                        if config.read(fullPath) != []:
+                            try:
+                                uuid_path = config.get('UUID_PATH', 'uuid_path')
+                                uuid_config = ConfigParser.RawConfigParser()
+                                uuid_config.add_section('GATEWAY')
+                                uuid_config.set('GATEWAY', gw.res_name, gw.res_uuid)
+                                with open(uuid_path, 'w') as configfile:
+                                    uuid_config.write(configfile)
+                            except ConfigParser.ParsingError, err:
+                                log.error('Could not open log config file')
```
Comment (pmasrani) [inline liota/dcc/vrops.py]: Do you mean 'log config file' or 'uuid file'?

**Comment by KohliDev** [comment]
Thanks for the review comments have made the necessary changes. I had just used the old class, renamed the class name and variables.

**Comment by pmasrani** [comment]
Ship It

**Pull request merged**